ethereum/referencetests/src/main/java/org/hyperledger/besu/ethereum/referencetests/GeneralStateTestCaseSpec.java

```diff
@@ -120,6 +136,7 @@ public int hashCode() {
   }
 
   /** Represents the "post" part of a general state test json _for a specific hard-fork_. */
+  @JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Why is this needed?

Reply:
The latest test artifacts have transaction exception / failure expectations that we are not currently using.